Assets/Scripts/Plate/Plate.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Plate : MonoBehaviour
+{
+    [SerializeField] private AudioSource breakSound;
+    private bool isBreaking = true;
+    
+    public void OnCollisionEnter(Collision other)
+    {
+        if (isBreaking)
```

Comment:
you need to invert it, if (isBreaking == false) then isBreaking = true and start coroutine, at the end of the coroutine is breaking would be false again